chsdi/static/doc/source/releasenotes/index.rst

```diff
@@ -27,6 +27,10 @@ API & applications
 `API <//api3.geo.admin.ch>`__
 '''''''''''''''''''''''''''''
 - Bug fixes
+- Announcement: the layer ch.babs.kulturgueter-anhoerung has been removed from FSDI services as previously announced
+- Announcement: the layer ch.bakom.downlink3 has been removed from FSDI services as previously announced
```

Comment:
@consta-v is this correct that it is added here already? I did not do anything yet for this decommissioning as you told me I can take a look later.

Reply:
@marionb I have taken this layer from prod to test for the deploy, next real update on the automat will be somewhen in summer, so we have no hurries here.

Reply:
Correct